projects/demo-playwright/tests/kit/input/input.spec.ts

```diff
@@ -0,0 +1,132 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe('Input', () => {
+    test("has custom content (text) + cleaner + hint which don't overlapping each others", async ({
+        page,
+    }) => {
+        await tuiGoto(
+            page,
+            '/components/input/API?tuiMode=null&tuiTextfieldCustomContent=<span>LongTextContent<%2Fspan>&tuiHintContent=Some%20content',
+        );
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '01-input.png',
+        );
+    });
+
+    test('correctly aligns single custom content (as large icon)', async ({page}) => {
+        await tuiGoto(
+            page,
+            '/components/input/API?tuiTextfieldCustomContent=tuiIconCalendarLarge',
+        );
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '02-input.png',
+        );
+    });
+
+    test("custom content (as large icon) + cleaner + hint don't overlapping each others", async ({
+        page,
+    }) => {
+        await tuiGoto(
+            page,
+            '/components/input/API?tuiTextfieldCleaner=true&tuiTextfieldCustomContent=tuiIconSearchLarge&tuiHintContent=Some%20content',
+        );
+
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '03-input.png',
+        );
+    });
+
+    test("custom content (as normal-size icon) + cleaner + hint don't overlapping each others", async ({
+        page,
+    }) => {
+        await tuiGoto(
+            page,
+            'components/input/API?tuiTextfieldCleaner=true&tuiTextfieldCustomContent=tuiIconVisaMono&tuiHintContent=Some%20content',
+        );
+
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '04-input.png',
+        );
+    });
+
+    test('input overflow due to placeholder', async ({page}) => {
+        await tuiGoto(
+            page,
+            'components/input/API?tuiTextfieldIconLeft=tuiIconSearch&pseudoFocus=true&placeholder=Lorem%20ipsum%20dolor%20sit%20amet,%20consectetur%20adipiscing%20elit,%20sed%20do%20eiusmod%20tempor%20incididunt%20ut%20labore',
+        );
+
+        const example = new TuiDocumentationPagePO(page).apiPageExample;
+        const input = example.locator('.t-input-wrapper').locator('input[tuiTextfield]');
```

Comment:
Does example contains only one `input[tuiTextfield]` ?

Could we shorten this line in the following way?
```suggestion
        const input = example.locator('input[tuiTextfield]');
```

Reply:
fixed